charts/argo-workflows/scripts/update-image-tag.sh

```diff
@@ -17,6 +17,14 @@ change_image_tag() {
   CHANGED=true
 }
 
+add_image_deployment_tag() {
+  current_image_tag="$IMAGE_TAG"
+  aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin "$ECR_REGISTRY"
+  MANIFEST=$(aws ecr batch-get-image --repository-name "${REPO_NAME}" --image-ids imageTag="${current_image_tag}" --region eu-west-1 --query 'images[].imageManifest' --output json)
+  # Every image that has a tag will then have 'deployed-to-production' tag added as well.
+  aws ecr put-image --repository-name "${REPO_NAME}" --image-tag "deployed-to-production" --image-manifest "$MANIFEST"
```

Comment:
We have to tag images being used in staging and integration also

```suggestion
  aws ecr put-image --repository-name "${REPO_NAME}" --image-tag "deployed-to-${ENVIRONMENT}" --image-manifest "$MANIFEST"
```